packages/flutter_tools/templates/app_shared/android-kotlin.tmpl/app/build.gradle.kts.tmpl

```diff
@@ -16,7 +16,7 @@ android {
     }
 
     kotlinOptions {
-        jvmTarget = JavaVersion.VERSION_1_8
+        jvmTarget = "1.8"
```

Comment:
Should this also be `JavaVersion.VERSION_1_8.toString()`, like the other instance? I'd also like to avoid hardcoded strings here

Reply:
Yup you're right. I missed it. I'll fix it tomorrow.

Feel free to push to my branch though and merge it.